src/common/client_response.hpp

```diff
@@ -48,7 +48,7 @@ constexpr T GetEnumFromPhaseIndex(size_t index) {
 //and YES phase can have 0 responses
 //every enum must have "_first" and "_last"
 //"_first" ==  "previous_enum::_last" + 1
-//EVERY response shall have unique ID (so every button in GIU is unique)
+//EVERY response shall have unique ID (so every button in GUI is unique)
```

Comment:
when you are fixing the comment wording, please make sure it is proper English too ;) 
... shall have **a** unique ID